power-wheel-card.js

```diff
@@ -618,6 +619,7 @@ class PowerWheelCard extends LitElement {
     this.autoToggleView = config.initial_auto_toggle_view;
     this.sensors = this._getSensors(config);
     this.view = config.initial_view;
+    // todo: remove this second assign
     this.config = Object.assign({}, config);
```

Comment:
Since config isn't the same object anymore, you could just replace this with `this.config = config;` now.

Reply:
Fixed on `dev` branch in version 0.1.0c.